Fenetres/CTabView.cpp

```diff
@@ -31,7 +31,7 @@ void CTabView::Select(int32	tab)
 	BRect	size;
 
 	// recuperer les vues des 2 onglets
-	selectedTab = TabAt(0);
+	selectedTab = TabAt(Selection());
```

Comment:
This change makes the app crash on launch.

Reply:
@humdingerb Oops. I don't know why I reverted my changes.
Will change it back.